src/modules/reactions-gif.js

```diff
@@ -1,22 +1,35 @@
 const sample = require('lodash/sample');
-const { gifs } = require('./reaction-list.json');
+const gifs = require('./reaction-list.json');
 const { SOCIAL_PREFIX } = require('../config');
 
 const reactionSocial = async function (response, { id, input, i18n }) {
     if (input[0] !== SOCIAL_PREFIX) {
         return response;
     }
 
-    const idWho = input.split(' ')[1].substr(2, input.length - 9);
+    if (input.indexOf(' ') > 0) {
```

Comment:
It is better to watch for  `mentions`  in discord message. But fine for now.
But.
Let's split once and work with results, it's dirty now. Smthng like:
```
let [reactionName, idWho] = input.split(' ');
if(idWho) {...}
```
slices and substrs in way you use them are unreadable for me =\